src/model/LabyrinthList.java

```diff
@@ -1,81 +0,0 @@
-package model;
-
-import java.io.File;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Scanner;
-
-public class LabyrinthList extends LabyrinthImpl {
-
-	private HashMap<Position, Integer> list;
-
-	public LabyrinthList(String path) {
-		try {
-			Scanner sc = new Scanner(new File(path));
-			startValue = sc.nextInt();
-			finishValue = sc.nextInt();
-			freeValue = sc.nextInt();
-			wallValue = sc.nextInt();
-
-			rows = sc.nextInt();
-			columns = sc.nextInt();
-
-			int objCount = sc.nextInt();
-			list = new HashMap<Position, Integer>();
-
-			for (int i = 0; i < objCount; i++) {
-				int x, y, type;
-				x = sc.nextInt();
-				y = sc.nextInt();
-				type = sc.nextInt();
-				list.put(new Position(x, y), type);
-				if (type == startValue)
-					start = new Position(x, y);
-				if (type == finishValue)
-					finish = new Position(x, y);
-			}
-			sc.close();
-
-		} catch (Exception ex) {
-			System.out.println("File was not found");
-		}
-	}
-
-	@Override
-	public int getRowCount() {
-		return rows;
-	}
-
-	@Override
-	public int getColumnCount() {
-		return columns;
-	}
-
-	@Override
-	public boolean isFreeAt(Position p) {
-		return !list.containsKey(p);
-	}
-
-	@Override
-	public boolean isWallAt(Position p) {
-		Integer value = list.get(p);
-		return value != null && value != startValue && value != finishValue;
-	}
-
-	@Override
-	public List<Position> getNeighbours(Position current) {
-		List<Position> neighbours = new ArrayList<Position>();
-		if (current.x - 1 >= 0)
```

Comment:
even if the "if condition" contains just one single instruction is a best practice that this instruction to be wrapped into curly brackets.